src/mds/objects.py

```diff
@@ -119,6 +122,68 @@ async def create_object(
     return JSONResponse(response, HTTP_201_CREATED)
 
 
+@mod.get("/objects/{guid:path}")
+async def get_object(
+    guid, request: Request,
```

Comment:
guid could use a type hint and we could type hint the return value as well with the `->` syntax